assets/scss/core/_variables.scss

```
@@ -78,20 +78,17 @@ $dp-text-aling: center;
   /* backgrounds grid, cebra */
 }
 /* TEXT COLORS */
-
 .dark-grey-txt {
   color: $dp-color-darkgrey;
   /* variant of <p> */
   font-weight: $dp-font-weight-bold;
 }
 /* BORDER COLORS */
-
 .sylver {
```

"Silver" de plateado? Es con "Y" griega.

---

Ok thanks!

---

Osea, habrái que cambniarlo asi?

```suggestion
.silver {
```